### ISSUE TITLE
1.20 release

### ISSUE THREAD

**Issue by blairconrad** [FakeItEasy/FakeItEasy#283]
**Ready** when all other issues forming part of the release are **Done**.
- [ ] run code analysis in VS in _Release_ mode and address violations (send a regular PR which must be merged before continuing)
- [ ] check build, create release in [GitHub UI](https://github.com/FakeItEasy/FakeItEasy/releases) including releaseNotes, mentioning non-owner contributors, if any
- [ ] push NuGet package
- [ ] copy release notes from GitHub to NuGet
- [ ] de-list pre-release NuGet packages if present
- [ ] update website with contributors list (if in place)
- [ ] tweet, mentioning contributors and post link as comment here for easy retweeting ;-)
- [ ] post tweet in JabbR ([fakeiteasy](https://jabbr.net/#/rooms/fakeiteasy) and [general-chat](https://jabbr.net/#/rooms/general-chat)) and Gitter ([FakeItEasy/FakeItEasy](https://gitter.im/FakeItEasy/FakeItEasy))
- [ ] post links to the NuGet and GitHub release in each issue in this milestone, with thanks to contributors
- [ ] use `rake set_version[new_version]` to change CommonAssemblyInfo.cs to expected minor version (of form _xx.yy.zz_)
- [ ] push to origin branch, create PR to upstream master
- [ ] use `rake create_milestone[new_version]` to
  - create a new milestone for the next release
  - create new issue (like this one) for the next release, adding it to the new milestone
- [ ] close all issues on this milestone
- [ ] close this milestone
